data/transactions/logic/eval_test.go

```diff
@@ -896,6 +896,15 @@ func TestBytecTooFar(t *testing.T) {
 	testPanics(t, "byte 0x23; bytec_1; btoi", 1)
 }
 
+func TestManualCBlockEval(t *testing.T) {
+	// TestManualCBlock in assembler_test.go demonstrates that these will use
```

Comment:
@jannotti Minor:  Is there a reason to omit the partition + parallel stanza here?

```suggestion
	partitiontest.PartitionTest(t)

	t.Parallel()

	// TestManualCBlock in assembler_test.go demonstrates that these will use
```

Reply:
Just general incompetence.